meta-mender-core/classes/mender-dataimg.bbclass

```diff
@@ -35,6 +35,78 @@ IMAGE_CMD_dataimg_mender-image-ubi() {
     install -m 0644 "${WORKDIR}/data.ubifs" "${IMGDEPLOYDIR}/${IMAGE_NAME}.dataimg"
 }
 
+do_generate_bootstrap_artifact() {
```

Comment:
Straight copy-pasta from the ArtifactImg generation.

I think it's okay, but can unify this (somewhow), if you guys feel it's needed.